rules/dispatch.go

```diff
@@ -102,112 +113,69 @@ func (r *DefaultRuleRenderer) hostDispatchChains(
 	}
 
 	if fromOnly {
-		return r.dispatchChains(
+		return r.dispatchEndpointChains(
 			names,
-			epMarkMapper,
 			HostFromEndpointPfx,
 			"",
 			ChainDispatchFromHostEndpoint,
 			"",
 			false,
 			false,
-			false,
 		)
 	}
 
 	if !applyOnForward {
-		return r.dispatchChains(
+		return r.dispatchEndpointChains(
 			names,
-			epMarkMapper,
 			HostFromEndpointPfx,
 			HostToEndpointPfx,
 			ChainDispatchFromHostEndpoint,
 			ChainDispatchToHostEndpoint,
 			false,
 			false,
-			false,
 		)
-
 	}
 
 	return append(
-		r.dispatchChains(
+		r.dispatchEndpointChains(
 			names,
-			epMarkMapper,
 			HostFromEndpointPfx,
 			HostToEndpointPfx,
 			ChainDispatchFromHostEndpoint,
 			ChainDispatchToHostEndpoint,
 			false,
 			false,
-			false,
 		),
-		r.dispatchChains(
+		r.dispatchEndpointChains(
 			names,
-			epMarkMapper,
 			HostFromEndpointForwardPfx,
 			HostToEndpointForwardPfx,
 			ChainDispatchFromHostEndPointForward,
 			ChainDispatchToHostEndpointForward,
 			false,
 			false,
-			false,
 		)...,
 	)
 }
 
-func (r *DefaultRuleRenderer) dispatchChains(
+func (r *DefaultRuleRenderer) dispatchEndpointChains(
```

Comment:
How bout calling this `interfaceNameDispatchChains`, to emphasise that they match on the name?

Reply:
Good point.